dandi/metadata.py

```diff
@@ -230,7 +230,7 @@ def nwb2asset(nwb_path, digest=None, digest_type=None):
     asset.wasGeneratedBy = models.Activity(
         identifier=str(uuid4()),
         name="NWB conversion",
-        description="Data converted using DANDI cli",
```

Comment:
This name again should be adjusted, see my other comment on making it into a list